src/clib/lib/enkf/obs_vector.cpp

```diff
@@ -460,12 +458,68 @@ obs_vector_type *obs_vector_alloc_from_GENERAL_OBSERVATION(
     }
 }
 
+static bool history_init_ts(const history_source_type history,
```

Comment:
`ts:= time_series`? What is going on is really something like `read_history_from_ecl_summary` right?

Reply:
Probably, yes. It is conditional, read history or result from ecl_summary, depending on the value of history_source_enum.

Reply:
Changed name and added docstring.